cli/azd/pkg/apphost/testdata/TestAspireContainerGeneration-main.bicep.snap

```diff
@@ -81,8 +81,8 @@ output AZURE_LOG_ANALYTICS_WORKSPACE_NAME string = resources.outputs.AZURE_LOG_A
 output AZURE_CONTAINER_APPS_ENVIRONMENT_NAME string = resources.outputs.AZURE_CONTAINER_APPS_ENVIRONMENT_NAME
 output AZURE_CONTAINER_APPS_ENVIRONMENT_ID string = resources.outputs.AZURE_CONTAINER_APPS_ENVIRONMENT_ID
 output AZURE_CONTAINER_APPS_ENVIRONMENT_DEFAULT_DOMAIN string = resources.outputs.AZURE_CONTAINER_APPS_ENVIRONMENT_DEFAULT_DOMAIN
-output SERVICE_MY_SQL_ABSTRACT_VOLUME_PROJECTAPPHOST_VOLUME_TWO_NAME string = resources.outputs.SERVICE_MY_SQL_ABSTRACT_VOLUME_PROJECTAPPHOST_VOLUME_TWO_NAME
-output SERVICE_MYSQLABSTRACT_VOLUME_PROJECTAPPHOST_VOLUMEONE_NAME string = resources.outputs.SERVICE_MYSQLABSTRACT_VOLUME_PROJECTAPPHOST_VOLUMEONE_NAME
+output SERVICE_MY_SQL_ABSTRACT_VOLUME_PROJECTAPPHOSTVOLUMETWO_NAME string = resources.outputs.SERVICE_MY_SQL_ABSTRACT_VOLUME_PROJECTAPPHOSTVOLUMETWO_NAME
```

Comment:
Is this the expected naming syntax? It looks a little off.. Would we prefer `_` separators at all?

Reply:
it is the expected yes. 
It is a side effect of : https://github.com/Azure/azure-dev/pull/4286/files/4be3d0c087103452bff9c270a6bd21845cdf629b#r1785526924